src/state/gvMessages.ml

```diff
@@ -10,6 +10,13 @@ module Message = struct
     | Single p -> loc p
     | Group { pieces; _ } -> pieces |> List.filter_map loc |> List.enum |> Enum.get
 
+  let severity_to_bs_alert m = match m.severity with
```

Comment:
Why `bs_alert`?

Reply:
This function converts the severity of the warning to the string of a fitting bootstrap alert. It is used for displaying for example failures and successful warnings different colors without needing to hand-craft the css.